### ISSUE TITLE
Data source?

### ISSUE THREAD

**Issue by mgei** [aledallas/ictax-app#1]
Hi, where do you get the data from? 

Is it what one can download here? https://www.ictax.admin.ch/extern/en.html#/xml 

I'd like to use it for 2020.

**Comment by aledallas** [comment]
That is correct, the source is on the ictax link you posted. I used the listing with V2. 
Still haven't got time to update it with 2020, but I can release it over the weekend.

**Comment by mgei** [comment]
Might be cool to have the script included to generate the *.rds files from the official XML.

Anyway, thumbs up for this project. Will come handy when I have to do my tax declaration.

**Comment by aledallas** [comment]
you can now find the script for the extraction and transformation of fund data from the XML. It's called [ETL_ictax.R](https://github.com/aledallas/ictax-app/blob/main/ETL_ictax.R)

**Comment by mgei** [comment]
Thanks! Seems like due to the data size, one needs quite some patience when reading.

**Comment by aledallas** [comment]
Yes, for me the processing of the part of the payments (dividends) takes approx 1,5 / 2 hours
